draft-ietf-scitt-architecture.md

```diff
@@ -459,19 +455,21 @@ A Transparent Statement remains a valid Signed Statement, and may be registered
 
 Verifiable Data Structure:
 
-: a data structure which supports one or more proof types, such as "inclusion proofs" or "consistency proofs" (as defined in {{-COMETRE}}).
-
+: a data structure which supports one or more proof types, such as "inclusion proofs" or "consistency proofs", for Signed Statements as they are Registered to a Transparency Service.
+SCITT supports multiple Verifiable Data Structures and Receipt formats as defined in {{-COMETRE}}, accommodating different Transparency Service implementations.
+May be referenced as an Append-only log, indicating it's immutable nature.
 {: #mybody}
 
 # Definition of Transparency
 
 In this document, the definition of transparency is intended to build over abstract notions of Append-only Logs and Receipts.
 Existing transparency systems such as Certificate Transparency are instances of this definition.
+SCITT supports multiple Append-only log formats, as defined in {{-COMETRE}}, and uses Verifiable Data Structure throughout the document.
 
 A Signed Statement is an identifiable and non-repudiable Statement made by an Issuer.
 The Issuer selects additional metadata and attaches a proof of endorsement (in most cases, a signature) using the identity key of the Issuer that binds the Statement and its metadata.
 Signed Statements can be made transparent by attaching a proof of Registration by a Transparency Service, in the form of a Receipt.
-Receipts demonstrate inclusion of Signed Statements in the Append-only Log of a Transparency Service.
+Receipts demonstrate inclusion of Signed Statements in the Verifiable Data Structure of a Transparency Service.
```

Comment:
```suggestion
Receipts demonstrate inclusion of Signed Statements in the Append-only Log of a Transparency Service.
```

Reply:
I don't like this.

Reply:
Can you elaborate on which you don't like? The PR change or the Suggestion?

Reply:
Because I don't know what an "append only log is"... if it is a normative dependency, and it is not a VDS, what are its requirements beyond the requirements of a VDS?

Reply:
There's a logical sequence of statements, which is not a normative dependency but is important when discussing audit. I think we can keep this line that way though.